plugins/catalog-import/src/components/ImportEntityPage.tsx

```diff
@@ -57,19 +57,19 @@ export const ImportComponentPage = (opts: StepperProviderOpts) => {
 
   return (
     <Page themeId="home">
-      <Header title="Register an existing component" />
+      <Header title="Register an existing entity" />
```

Comment:
I'm not sure if entity is the best naming-choice here, but I'm also not sure what to use instead. Is the end-user of a Backstage-based portal aware of the meaning of "entity"?

Reply:
I feel like the correct generic term that refers to a component, an API etc... is an entity based on Backstage's documentation as well as several places on this page that already referred to entities rather than components

Reply:
I would say entity if fine here on the page and in some other parts. Just the create PR part is designed only for components right now, we should keep the wording there.

Reply:
Btw: The link in the Scaffolder uses "Register Existing Component" to link to the catalog-import:
https://github.com/backstage/backstage/blob/118df55e723cb8cc398cad4705e7f39a350bd0ef/plugins/scaffolder/src/components/ScaffolderPage/ScaffolderPage.tsx#L134-L141

They should probably be aligned to the same wording to keep the "create something new / add something existing" storylines aligned.